solr/webapp/web/js/angular/controllers/index.js

```diff
@@ -29,6 +29,10 @@ solrAdminApp.controller('IndexController', function($scope, System, Cores, Const
         sessionStorage.setItem("auth.username", data.security.username);
       }
 
+      if ("authenticationPlugin" in data.security && data.security.authenticationPlugin != null) {
```

Comment:
I'd have to see if we ahve an established pattern for these if's!   `if data.security.authenticationPlugin ` migth work...

Reply:
Yes, that worked for me